README.md

```diff
@@ -475,6 +475,9 @@ Other Style Guides
     // bad
     const name = "Capt. Janeway";
 
+    // bad
```

Comment:
should this elaborate, and say "because template literals should always contain an interpolation"?